source/release-notes/1.6.txt

```diff
@@ -13,6 +13,21 @@ Release Notes for mongosync 1.6
    :depth: 2
    :class: singlecol
 
+Patch Releases
+--------------
+
+.. _1.6.1-c2c-release-notes:
+
+1.6.1 Release
+~~~~~~~~~~~~~
+
+Issues Fixed:
+
+- Fixed a bug that may cause long running ``mongosync`` operations to
+  prematurely timeout. In consequence of this change, ``mongosync`` produces
```

Comment:
```suggestion
  prematurely timeout. Now, ``mongosync`` produces
```
[Minimalism]
Considering shortening the phrase "In consequence of" for minimalism.